### PR TITLE
Support for protocol 80949 (v5.0.0)

### LINKED ISSUE #121: Patch 5.0 Broke replay loading
Loading replays I get

`unpack requires a buffer of 4 bytes`

on line 91 in decoders.py.

### PR DESCRIPTION
**Pull request by Talv**
Fixes #121

### CODE REVIEW

File: sc2reader/resources.py [hunk 1]
```diff
@@ -775,6 +775,11 @@ def register_default_readers(self):
             readers.GameEventsReader_65895(),
             lambda r: 65895 <= r.base_build,
```
Comment (StoicLoofah) [inline sc2reader/resources.py]: Can you update this line as well to have a max build to be in line with the others above?

File: sc2reader/resources.py [hunk 2]
```diff
@@ -867,6 +872,10 @@ def register_default_datapacks(self):
             datapacks["LotV"]["77379"],
             lambda r: r.expansion == "LotV" and 77379 <= r.build,
```
Comment (StoicLoofah) [inline sc2reader/resources.py]: Same here

**Review by StoicLoofah — CHANGES_REQUESTED** [review]
It looks like the build for styles on `reader.py` isn't passing. Did you run black on that file?

Thanks for tackling this so quickly!

**Comment by Talv** [comment]
Should be good now. And yeah I forgot about `black`.

**Review by StoicLoofah — APPROVED** [review]
Great work. Thanks so much! I'll merge this in and mark a new release version

**Pull request merged**